### PR TITLE
adding AuxTel cabinet content - tickets/SITCOM-1643

### PR DESCRIPTION
**Pull request by Garavena89**
Hi all!
Just added this ticket about migrating the cabinets content in AuxTel.
I had the idea of adding breath description or links to each component for more details about what they are but I believe not all of them are already uploaded in obs-docs. Also it has a Story points of 1, so I think there is no need for more details. Nevertheless, if all are agree with that I can attempt to add them in corrections.

Cheers!

### CODE REVIEW

**Review by isotuela — CHANGES_REQUESTED** [review]
Dear Gonzalo, 
Thank you for this. I added three comments:
- Author is Manuel Gomez 
- All images must have self-contained descriptive captions for easier navigation. 

I will leave the PR in Request Changes to finally approve once you squash all commits into one. Thanks again!

File: AuxTel/Components/Cabinet-Content-Diagrams.rst
Comment (Garavena89) [reply AuxTel/Components/Cabinet-Content-Diagrams.rst:46]: great idea Ioana, thx

**Review by isotuela — APPROVED** [review]
Thank you for all the changes! It looks great. 

Remember to squash all commits before merging. 

Cheers, 
Ioana

**Pull request merged**
